booking_tool.py

```diff
@@ -98,7 +98,7 @@ def book(args):
                     logging.info('booked earliest')
 
                     # Continue to book
-                    handlers.continue_to_book(driver=driver)
+                    handlers.continue_to_book(driver=driver,golfers=config.golfers)
```

Comment:
This means that we need to make `num-golfers` a required flag.

Reply:
probably makes sense to do